content/applications/inventory_and_mrp/inventory/shipping/operation/multipack.rst

```diff
@@ -1,84 +1,117 @@
+   top-right corner of the delivery order. Click the :guilabel:`Packages` smart button to go to the
+   :guilabel:`Packages` page for the delivery order, where each package can be selected to view all
+   of the items included in it.
 
-It will automatically create a pack with the requested quantity.
+   .. image:: multipack/packages-smart-button.png
+      :align: center
+      :alt: The Packages smart button on a delivery order.
 
-Do the same steps for the other pieces you want to pack in.
+Create a backorder for items to be shipped later
+================================================
 
-.. image:: multipack/multipack04.png
-   :align: center
+If some items will be shipped at a later date than others, there is no need to put them in a package
+until they are ready to be shipped. Instead, create a backorder for the items being shipped later.
+
+Begin by shipping the items that will be shipped immediately. If they will be shipped in multiple
+packages, follow the :ref:`steps above <inventory/shipping/multiple-packages>` to package them as
+required. If they will be shipped in a single package, simply mark in the :guilabel:`Done` column
+the quantity of each item being shipped, but **do not** click the :guilabel:`Put In Pack` button.
 
-Click on **Validate** when you are done.
+After all quantities being shipped immediately are marked as :guilabel:`Done`, click the
+:guilabel:`Validate` button, and a :guilabel:`Create Backorder?` pop-up window appears. Then, click
+the :guilabel:`Create Backorder` button. Doing so confirms the items being shipped immediately and
+creates a new delivery order for the items that will be shipped later.
 
-Multi-packages with a 3rd party-shipper
----------------------------------------
+.. image:: multipack/backorder-pop-up.png
+   :align: center
+   :alt: The Create Backorder? pop-up window.
 
-Once the delivery order is validated, you will receive the tracking
-number. The **carrier Tracking ref** field will automatically be filled.
-Click on the **Tracking** button to check your delivery on the provider
-website.
+The backorder delivery order will be listed in the chatter of the original delivery order in a
+message that reads :guilabel:`The backorder WH/OUT/XXXXX has been created.`. Click on
+:guilabel:`WH/OUT/XXXXX` in the message to view the backorder delivery order.
 
-.. image:: multipack/multipack05.png
+.. image:: multipack/backorder-chatter.png
    :align: center
+   :alt: The backorder delivery order listed in the chatter of the original delivery order.
 
-The **labels** to stick on your packages are available in the history
-underneath:
+The backorder delivery order can also be accessed by navigating to :menuselection:`Inventory`,
+clicking the :guilabel:`X Back Orders` button on the :guilabel:`Delivery Orders` card, and selecting
+the delivery order.
 
-.. image:: multipack/multipack01.png
+.. image:: multipack/back-orders-button.png
```

Comment:
Optional: this is more useful if the other items on the UI that surround this button/section are included. Those other elements help act as directional queues for the reader.

Reply:
The other sections on the Inventory overview page differ depending on the apps installed and settings enabled for each user. This also means that the Delivery Orders card will appear in a different location for some users, so I think it makes more sense to focus on the single card we're trying to point out, rather than its location on the page.